phpunit.xml

```diff
@@ -15,10 +15,17 @@
     </testsuites>
 
     <logging>
-        <log type="coverage-html" target="tests/coverage/"/>
+        <log type="coverage-clover" target="build/logs/clover.xml"/>
```

Comment:
I found this article: http://mnapoli.fr/code-coverage-reports-with-github-travis-and/
It seems like we can get rid of the logging part alltogether (since it's annoying to have the reports generated everytime the unit tests are run) and only update the travis.yml file.
It seems like a better solution. What do you think?